### ISSUE TITLE
Devkit Ground Upgrade doesn't work on Insane sized maps

### ISSUE THREAD

**Issue by NoThreat** [SmartlyDressedGames/Unturned-3.x-Community#1551]
When you got things done, selected materials, pressed "Upgrade" - it doesn't work. Foliage_Volume creates as usual, but then you save map and after opening editor - nothing were saved. No foliage volume, no materials, nothing. 

(Used vanilla materials, no tiles were missed)

**Comment by SDGNelson** [comment]
Using the upgrade wizard should not be creating a foliage volume. I will add the "next patch" label as a reminder to myself to give this another try.

**Comment by SDGNelson** [comment]
Giving this a try I downloaded The Driftless map from the workshop. It looks like it has a layer of landscape underneath the legacy terrain so I deleted the level hierarchy to start with a clean slate. From there the upgrade process worked okay without issues.

To clarify the usage of the tool I wrote a page for it: https://github.com/SmartlyDressedGames/U3-Docs/blob/master/Landscape.md

In the next update I have made it a bit nicer for the big maps by time-slicing the upgrade process and using grass as the default unset material. For the meantime it should work perfectly fine for your map, so let me know if you still have problems.
